_includes/ref/bitcoin-core/rpcs/intro.md

```diff
@@ -9,153 +9,199 @@ http://opensource.org/licenses/MIT.
 
 {% autocrossref %}
 
-Bitcoin Core provides a large number of Remote Procedure Calls (RPCs)
-using a [HTTP JSON-RPC version 1.0][] interface. Any program can access
-the RPCs using JSON-RPC, but Bitcoin Core also provides the
-`bitcoin-cli` command to wrap the JSON-RPC access for Command Line
-Interface (CLI) users. Most of the RPC examples in this documentation
-use `bitcoin-cli` for simplicity, so this subsection describes raw
-JSON-RPC interface and how the command-line interface translates it.
-
-In order to start `bitcoind`, you will need to set a password for
-JSON-RPC in the `bitcoin.conf` file. See the [Examples
-Page][devexamples] for details. JSON-RPC starts on port 8332 for mainnet
-and 18332 for testnet and regtest. By default, `bitcoind` doesn't use a
-JSON-RPC user, but you can set one (see `bitcoind --help`).
-
-The interface is not intended for public access and is only accessible
-from localhost by default.
-
-RPCs are made using the standard JSON-RPC 1.0 syntax, which sends several
-standard arguments:
+Bitcoin Core provides a remote procedure call (RPC) interface for various
+administrative tasks, wallet operations, and queries about network and block
+chain data.
+
+If you start Bitcoin Core using `bitcoin-xt`, the RPC interface is disabled by
+default. To enable it, set `server=1` in `bitcoin.conf` or supply the `-server`
+argument when invoking the program.
+
+If you start Bitcoin Core using `bitcoind`, the RPC interface is enabled by
+default. To disable it, set `server=0` in `bitcoin.conf`.
```

Comment:
Does using bitcoind without server=1 make sense?  If you do that, there's no way to see what it's doing or to stop it using the `stop` command.  (It should stop safely if killed using the right signal, but that seems like something most people would want to avoid.)